### DISCUSSION TITLE
〔水帖〕这 简 必须抽

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#2677] [其他]
![1723772257149](https://github.com/user-attachments/assets/070e3c43-f799-410b-869f-74161f6cbb38)

**Comment by zhixuan45** [comment]
抽完朱鸢之后半个子都没了...

**Comment by HKLov3** [comment]
81抽的青衣、70抽的玉壶，已经被榨干了<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >

**Comment by panda114514667** [comment]
本来想抽赛斯的但是这角色动作后摇好长啊

**Comment by Moshuimk** [comment]
这图笑死

**Comment by I-ikun** [comment]
小黑子
